website/content/docs/running-staticcheck/ci/github-actions/index.md

```diff
@@ -37,27 +35,25 @@ on: ["push", "pull_request"]
 
 jobs:
   ci:
-	name: "Run CI"
-	strategy:
-	  fail-fast: false
-	  matrix:
-		os: ["windows-latest", "ubuntu-latest", "macOS-latest"]
-		go: ["1.16.x", "1.17.x"]
-	runs-on: ${{ matrix.os }}
-	steps:
-	- uses: actions/checkout@v1
-	  with:
-		fetch-depth: 1
-	- uses: WillAbides/setup-go-faster@v1.7.0
-	  with:
-		go-version: ${{ matrix.go }}
-	- run: "go test ./..."
-	- run: "go vet ./..."
-	- uses: dominikh/staticcheck-action@v1.2.0
-	  with:
-		version: "2022.1.1"
-		install-go: false
-		cache-key: ${{ matrix.go }}
+    name: "Run CI"
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["windows-latest", "ubuntu-latest", "macOS-latest"]
+        go: ["1.17.x", "1.18.x"]
```

Comment:
As it's for documentation purpose, I would suggest `["stable", " oldstable"]`

And if you disagree, to use 1.21.x and 1.22.x

Reply:
@ccoVeille I think using explicit version (1.21.x, 1.22.x) is better. I have applied the changes just now

Reply:
I'm OK with that.

On most project, and for most people, having fixed version is safer.

Some people are not aware of these, and if maintainer is not very active, every new PR could fail in the CI if something breaks the build by using a new version

Reply:
Maybe a yaml comment here, such as

# please adapt this to your need and recent version when available